Button positions in the Compose Editor

loma-one opened the pull request:
The position of the buttons below the editor were previously placed on the right. The sandwitch and the position information were separated from the buttons for preview and send and placed on the left and right respectively.

old
![2025-07-14_11-57](https://github.com/user-attachments/assets/a5d4644c-dda4-440a-a0ec-9c3c26829591)

new
![2025-07-14_11-59](https://github.com/user-attachments/assets/7d5d8325-b89a-4831-8578-5f1b087cddf4)

loma-one committed:
Button positions in the Compose Editor

Die Position der Button unter dem Editor waren bisher rechts plaziert. Das Sandwitch und die Positionsangabe wurden von den Button für die Vorschau und Senden getrennt und jeweils Link und Rechts gesetzt.

marcusxss commented on view/templates/item/compose.tpl:
```diff
@@ -115,7 +113,29 @@
+<style>
+.left-section {
+    float: left;
+    width: 50%;
```
Bootstrap has some builtin classes called pull-right and pull-left that applies float, which friendica does use in various places:
https://getbootstrap.com/docs/3.4/css/#helper-classes-floats

I think you can just remove text-align right on the box, add a box around what should be put to the right and add `pull-right` to it. Explicitly setting pull/float left shouldn't be necessary.

Also bootstrap has a class to clear floats, called `clearfix`:
https://getbootstrap.com/docs/3.4/css/#helper-classes-clearfix

loma-one replied:
Thank you very much for this tip. That is much simpler.  I hope it fits.

marcusxss replied:
You're welcome!

I think you may actually just be able to put the the things that should be to the right into a new container which has `text-align: right`, and then not have to deal with floats and clearing them.

marcusxss commented on view/templates/item/compose.tpl:
```diff
@@ -115,7 +113,29 @@
+    overflow: hidden; /* Clearfix to contain the floated elements */
+}
+
+.comment-edit-preview {
+    margin-top: 50px; /* Adjust the value as needed */
+}
```
What is this for?

loma-one committed:
Button positions in the Compose Editor

The position of the buttons below the editor were previously placed on the right. The sandwitch and the position information were separated from the buttons for preview and send and placed on the left and right respectively.

annando commented:
IS this really expected to be a PR against develop or should it be against the RC?

loma-one commented:
Falsches Repro

annando commented:
You can simply change the target branch in the pull request.
